### ISSUE TITLE
运行python代码后报错。。

### ISSUE THREAD

**Issue by auisydh** [CrazyBoyM/llama3-Chinese-chat#65]
你好！为什么运行python代码后会报错，提示：The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
Setting pad_token_id to eos_token_id:128001 for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
改完之后虽然没有报错了，但是大模型没有输入任何内容，只有我自己一行一行的输入，请问问题出在哪里呢？
